INSTALL

```diff
@@ -556,7 +560,7 @@ The specific libraries and their options are:
   same location.
   URL: http://crd-legacy.lbl.gov/~xiaoye/SuperLU
   Options: SUPERLU_OPT, SUPERLU_LIB.
-  Versions: SuperLU_DIST >= 5.1.0.
+  Versions: SuperLU_DIST >= 6.3.0.
```

Comment:
Shouldn't this still say 5.1.0?  We could add a comment that 6.3.0 or newer is recommended.

Reply:
Yup, I missed this one.  I'll fix it.